src/components/send-money/SendMoneyAmountInput.js

```diff
@@ -80,10 +81,9 @@ class SendMoneyAmountInput extends Component {
       }
       let amountAttoNear = ''
       if (value !== '') {
-         let input = new Big(value).times(new Big(10).pow(NOMINATION))
-         amountAttoNear = input.toFixed()
+         amountAttoNear = utils.format.parseNearAmount(value);
          let balance = new Big(this.props.amount)
-         if (balance.sub(input).s < 0) {
+         if (balance.sub(new Big(amountAttoNear)).s < 0) {
```

Comment:
if this is the only remaining use of `Big`, I think we can avoid it by using `BN` to do the same math (will reduce JS bundle size).

Reply:
I agree, it's a good follow up item. Would prefer to postpone to after this change ships to not block core.

Reply:
Small stuff like that tends to not get fixed for long time if not addressed immediately on the spot.
I've created PR https://github.com/nearprotocol/near-wallet/pull/275